utils/check_package_version_consistency.js

```diff
@@ -48,22 +58,25 @@ async function main() {
 
     let nbErrors = 0;
     // finding the packages that are present with multiple versions
-    for (const [module, versionPackages] of Object.entries(dependencies)) {
-        const versions = Object.keys(versionPackages);
-        if (versions.length !== 1) {
-            console.log("Warning module ", module, " has multiple versions ", versions.join(" "));
-            console.log(versionPackages);
-            nbErrors++;
+    for (const collectedDepedencies of [allDependencies, devDependencies]) {
+        for (const [module, versionPackages] of Object.entries(collectedDepedencies)) {
+            const versions = Object.keys(versionPackages);
+            if (versions.length !== 1) {
+                console.log("Warning module ", module, " has multiple versions ", versions.join(" "));
+                console.log(versionPackages);
+                nbErrors++;
+            }
```

Comment:
Hmm, just noticed that this check does not need to be performed for the `devDependencies` as well, I will adjust these lines.

Reply:
Fixed in 3c88f716f078632e34e1aabd570cc378c569dbb6